### ISSUE TITLE
Implement incremental backups of badgerauth nodes

### ISSUE THREAD

**Issue by amwolff** [storj/edge#166]
Depends on #163.

The blueprint has [a section](https://github.com/storj/gateway-mt/blob/3ef75f412a50118d9d910e1b372e126e6ffb7503/docs/blueprints/new-auth-database.md#backups) about handling backups, but it's not exhaustive. The implementation should also consider backing up to Storj DCS.

**Comment by shaupt131** [comment]
Incremental backups can be part of a backup chore that can be run at regular intervals.

**Comment by shaupt131** [comment]
Other points: When was the last backup? How are backups going to be organized (1 bucket for auth service backup snapshots with node ID? other?) How often are we doing to back up?

**Comment by halkyon** [comment]
We discussed that incremental backups may not be worth pursuing right now due to complexities of storing and restoring incremental backups, which requires a non-trivial investment in time to get right compared to full backups. New issue to run full backups: https://github.com/storj/gateway-mt/issues/184 which we could work on implementing first, and then do incremental at a later date.

**Comment by storjBuildBot** [comment]
Change https://review.dev.storj.io/c/storj/gateway-mt/+/7263 mentions this issue.

**Comment by ferristocrat** [comment]
^^^ Comment for when Michael asks the question in the future -- the above change was used to base the work for full backups https://github.com/storj/gateway-mt/issues/184 ... Incremental backlogs will remain in the backlog to potential pick up in the future
